osm-georeferencing/main.py

```diff
@@ -313,7 +305,7 @@ def get_geopick_auth():
     :return: Authorization header
     """
     auth_info = {
-        'username':  os.environ.get('GEOPICK_USER'),
+        'username': os.environ.get('GEOPICK_USER'),
```

Comment:
Did you test with getting these env's from the secret store?

Reply:
Good point. I've added these secrets to the secret store and updated the names of the secrets. 
To add the secrets, the mas will need to be updated in the orchestration service - i can add the secrets and check the deployment file. Not sure how to explicitly test without deploying the mas first though.

Reply:
Disscussed during daily